.travis.yml

```diff
@@ -31,6 +31,9 @@ before_install:
 install:
   - travis_retry mvn -B install clean -U -DskipTests=true
 
+after_success:
+  - mvn jacoco:report coveralls:report
+
```

Comment:
Also, not sure what will happen when Travis tries to do this from its test matrix. We should probably make sure it runs just once, not 4 times.